edb/lib/cal.edgeql

```diff
@@ -1177,6 +1415,141 @@ std::datetime_get(dt: cal::local_datetime, el: std::str) -> std::float64
 };
 
 
+CREATE FUNCTION
+std::duration_get(dt: cal::date_duration, el: std::str) -> std::float64
+{
+    CREATE ANNOTATION std::description :=
+        'Extract a specific element of input duration by name.';
+    SET volatility := 'Stable';
```

Comment:
Why `Stable`?

Reply:
You're correct, only the version dealing with `timestamptz` is marked as "stable", the rest are "immutable". Fixed.

Reply:
`timestamptz` version should also be `immutable`.  It is stable in Postgres because it has configurable timezones, EdgeDB doesn't.

Reply:
This is one of those examples where `edgedb.raise` is stable and it's part of the implementation here.

The `edgedb.raise` itself has its own reason why it's [stable](https://github.com/edgedb/edgedb/blob/5d7a2dd3fd72dcb183ec080648d652bf99582c4d/edb/pgsql/metaschema.py#L1023-L1029). Basically we don't want them to be optimized by Postgres in a way that causes it to be evaluated (thus raising the exception) because it uses a constant (NULL) input regardless of the actual query where it's used.